modules/jaggery-apps/portal/gadgets/user_auth_apps/js/gadget.js

```diff
@@ -15,10 +15,18 @@ function drawPage() {
     if (json != null) {
 
         if (isArray(json.return)) {
+
             for (var i in json.return) {
+
+                var username = json.return[i].username;
+
+                if (username.indexOf("carbon.super") > -1) {
```

Comment:
carbon.super should be available as a constant. Can re-use it. So how would this work in tenant mode? There are util methods available to getTenantAwareUserName to seperate user name from tenant domain.